versioned_docs/version-6.x.x/core-concepts/validations-and-constraints.md

```diff
@@ -139,17 +136,17 @@ sequelize.define('foo', {
 });
 ```
 
-Note that where multiple arguments need to be passed to the built-in validation functions, the arguments to be passed must be in an array. But if a single array argument is to be passed, for instance an array of acceptable strings for `isIn`, this will be interpreted as multiple string arguments instead of one array argument. To work around this pass a single-length array of arguments, such as `[['foo', 'bar']]` as shown above.
+If you need to pass multiple arguments to the built-in validation functions, pass an array. But if a single array argument is to be passed, for example an array of acceptable strings for `isIn`, this is interpreted as multiple string arguments instead of one array argument. To work around this, pass a single-length array of arguments, such as `[['foo', 'bar']]`.
 
-To use a custom error message instead of that provided by [validator.js](https://github.com/chriso/validator.js), use an object instead of the plain value or array of arguments, for example a validator which needs no argument can be given a custom message with
+To use a custom error message instead of built-in validation methods provided by [validator.js](https://github.com/chriso/validator.js), use an object instead of the plain value or array of arguments. For example, include a validator with no argument as a custom message:

```

Comment:
```suggestion
To use a custom error message instead of built-in string validators provided by [validator.js](https://github.com/chriso/validator.js), use an object instead of the plain value or array of arguments. For example, include a validator with no argument as a custom message:

```